data/ARYAN-CODES-STAR.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Aryan Sharma",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Coder, Blogger, college student",
+  "avatar": "https://avatars.githubusercontent.com/u/78686630?s=400&u=f91c674137ab099d4fe15b58687eea1c77c633da&v=4",
```

It's best to use this link as it's shorter and updates automatically. 🙂
```suggestion
  "avatar": "https://github.com/ARYAN-CODES-STAR.png",
```